Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM python:3.5
+ENV PYTHONUNBUFFERED 1
+RUN mkdir /code
+WORKDIR /code
+ADD requirements.txt /code/
+RUN pip install -r requirements.txt
```

Comment:
Actually, we need to test the `COPY` and `volume` together, so we can have a deploy and development environment in the same place.